### ISSUE TITLE
Add support for CameraForCoordinateBounds

### ISSUE THREAD

**Issue by DincaAndrei2410** [tuyen-vuduc/mapbox-maui#26]
Besides CameraOptions for moving camera to a certain Center point, would be nice to expose a way for fitting some coordinate bounds. 

Android native skd has the following method defined: https://docs.mapbox.com/android/maps/api/11.1.0/mapbox-maps-android/com.mapbox.maps/-mapbox-map/camera-for-coordinate-bounds.html?query=open%20override%20fun%20cameraForCoordinateBounds(bounds:%20CoordinateBounds,%20boundsPadding:%20EdgeInsets?,%20bearing:%20Double?,%20pitch:%20Double?,%20maxZoom:%20Double?,%20offset:%20ScreenCoordinate?):%20CameraOptions

**Comment by tuyen-vuduc** [comment]
@DincaAndrei2410 Check out [this usage](https://github.com/tuyen-vuduc/mapbox-maui/blob/49e92120773898719971099f78df53a96fcf4669/src/qs/MapboxMauiQs/Examples/Annotations/58.AnimatePointAnnotation/AnimatePointAnnotationExample.cs#L66).

**Comment by DincaAndrei2410** [comment]
I've checked it. Rather then getting coordinates for camera bounds, I've did temporary something like the following on Android:

public void MoveCameraInBounds(Bounds bounds, double paddingFromEdges = 25)
    {
        var southWestPoint = Point.FromLngLat(bounds.BottomLeftLocation.Longitude, bounds.BottomLeftLocation.Latitude);
        var northEastPoint = Point.FromLngLat(bounds.TopRightLocation.Longitude, bounds.TopRightLocation.Latitude);

        var options = _mapView.MapboxMap.CameraForCoordinateBounds(
            new CoordinateBounds(southWestPoint, northEastPoint),
            new EdgeInsets(paddingFromEdges, paddingFromEdges, paddingFromEdges, paddingFromEdges),
            null, null, null, null);
        
        var animationOptions = MapAnimationOptions.CompanionField.MapAnimationOptions(
            new Function1Action<MapAnimationOptions.Builder>((builder) =>
            {
                builder.Duration(1000);
            }));

        CameraAnimationsUtils.FlyTo(
            _mapView.MapboxMap,
            options,
            animationOptions,
            null);
    }
    
    
Basically, for given geometry bounds and some margin, make sure that camera moves to show it

**Comment by tuyen-vuduc** [comment]
@DincaAndrei2410 Do you mean you want another method similar to `CameraForCoordinateBouds`?

**Comment by DincaAndrei2410** [comment]
Yeah, a shared method that gets southWestPoint, northEastPoint and edge insets. 
What I want to do with this method is to give those 2 points from a geometry and edge insets and the effect should be camera movement to fit that shape in its viewport

**Comment by tuyen-vuduc** [comment]
New version will be available https://www.nuget.org/packages/Mapbox.Maui/11.5.1-alpha03

**Comment by tuyen-vuduc** [comment]
Example usage
```
var bounds = map.MapboxController.GetCoordinateBoundsForCamera(map.CameraController.CameraState);
IEnumerable<IPosition> coords = new IPosition[] {
    bounds.Southwest.ToPoint().Coordinates,
    bounds.Northeast.ToPoint().Coordinates
};
map.MapboxController.CameraForCoordinates(
    coords
    , completion: (x) =>
     {

     });
 ```

**Comment by tuyen-vuduc** [comment]
@DincaAndrei2410  Have you checked this out?
